.github/workflows/code-quality.yml

```
@@ -81,12 +81,12 @@ jobs:
     - name: 🔍 Run PyLint
       run: |
         echo "🔍 Running PyLint analysis..."
-        pylint backend/src/ --output-format=text --reports=y || true
+        pylint backend/backend/src/ --output-format=text --reports=y || true
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)